lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -308,34 +319,30 @@ sub _register_plugins {
 }
 
 sub _render {
-  my ($renderer, $c, $output, $options) = @_;
-
-  # fallback to default renderer
-  unless (exists $c->stash->{openapi}) {
-    my $renderer = $c->app->renderer;
-    my $handler  = $renderer->handlers->{$renderer->default_handler};
-    $c->app->log->debug(
-      "Using default_handler to render data since 'openapi' was not found in stash. Set 'handler' in stash to avoid this message."
-    );
```

Comment:
I hope this message is not required after doing the change in `_before_render`.

@nicomen: I wonder if the reason why you saw this message was because of a change in `render_maybe()`, though I can't figure out what the change was.